### PR TITLE
Update mkdocs.yml

### PR DESCRIPTION
**Pull request by patrickfletcher**
add LaTeX support to docs via KaTeX

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -39,3 +39,15 @@ plugins:
             show_source: true
             show_signature_annotations: true
 
+#https://squidfunk.github.io/mkdocs-material/reference/math/
+markdown_extensions:
```
Comment (WolfByttner) [inline mkdocs.yml:43]: Do these have to be installed? We might need to add them to the requirements.txt file before merging

**Review by WolfByttner — APPROVED** [review]
Just check if we need to install something (and add it to requirements.txt) before merging!

Looks like we need this:
https://facelessuser.github.io/pymdown-extensions/installation/

**Comment by patrickfletcher** [comment]
Yeah I'm not sure - is there a way to test the output docs at the PR level? like a temp version of the Docs site?

File: mkdocs.yml (hunk 1 above)
Comment (patrickfletcher) [reply mkdocs.yml:43]: will add it.

**Comment by patrickfletcher** [comment]
@WolfByttner I'm not sure how to test this without merging to main.  Would be nice to have a staging area for Docs before they go live on main...

RE: LaTeX, I was aiming to have basic/simple/fast LaTeX support: nothing fancy is needed.

**Comment by WolfByttner** [comment]
@patrickfletcher  There is a way to test it, but it involves changing how docs are deployed in the project settings. You should be able to change which branch docs are deployed from temporarily. Alternatively, set up a clean venv with no system libraries, then install the requirements in the requirements file and run mkdocs build.

**Comment by patrickfletcher** [comment]
OK, that makes sense. Should be able to build them locally.

**Review by WolfByttner — APPROVED** [review]

**Pull request merged**